### ISSUE TITLE
No such file error when deploying Che with DevWorkspace operator

### ISSUE THREAD

**Issue by l0rd** [eclipse-che/che#19315]
### Describe the bug

Installation of nightly Che with devworkspace enabled fails. 

This is the error message I see in the Che operator logs: 

```
time="2021-03-17T16:39:27Z" level=info msg="Creating a new object: CustomResourceDefinition, name devworkspacetemplates.workspace.devfile.io"
time="2021-03-17T16:39:27Z" level=error msg="open /tmp/devworkspace-operator/templates/deployment/openshift/objects/workspaceroutings.controller.devfile.io.CustomResourceDefinition.yaml: no such file or directory"
```

### Che version

nightly

### Steps to reproduce

Deploy Che a fresh OCP 4.7.1 cluster with `chectl` and a CR patch to enable devworkspace.

### Runtime

OpenShift 4.7.1

### Installation method

```
$ chectl version
chectl/0.0.20210317-next.6e24ea8 darwin-x64 node-v12.21.0
$ cat ./patch.yaml
spec:
  devWorkspace:
    enable: true
$ chectl server:deploy --che-operator-cr-patch-yaml=patch.yaml -p openshift --no-olm-suggested-namespace -n workspaces-server
```

### Environment

OpenShift Dev Cluster

**Comment by flacatus** [comment]
@l0rd Fix provided here: https://github.com/eclipse-che/che-operator/pull/721
After che operator image build the problem will dissapear.

**Comment by l0rd** [comment]
@flacatus thanks it works now. I have a problem with the stable channel though. I get a different error in the operator logs:

```
time="2021-03-17T17:24:36Z" level=error msg="Error uninstalling Image Puller: no matches for kind \"KubernetesImagePuller\" in version \"che.eclipse.org/v1alpha1\""
``` 

Do you already have an issue for that or should I create one?

**Comment by tolusha** [comment]
@l0rd 
It is fixed in the master branch and backported in 7.27.x branch.
Will be fixed in 7.27.2

**Comment by l0rd** [comment]
@tolusha thanks, closing this issue then as everything works in master now.
